rdrf/rdrf/dynamic_data.py

```diff
@@ -714,6 +692,9 @@ def save_dynamic_data(self, registry, collection_name, form_data, multisection=F
             # create context_model NOW  to get context_id
             # CREATE MODE is used ONLY by multiple context form groups to enable cancellation in GUI
             context_id = self._create_context_model_on_fly()
+            # we've refactored the ClinicalData object so that context_id is now on the model:
+            record.context_id = context_id
+            # keeping this line for backward compatibility for now 
```

Comment:
What does "backward compatibility" mean in this context?

Reply:
Good question - I think I was thinking about reports - I'll have a look at them now to check